modmesh/profiling/_profiling_result_printer.py

```diff
@@ -0,0 +1,163 @@
+# Copyright (c) 2025, Han-Xuan Huang <c1ydehhx@gmail.com>
```

Comment:
As I mentioned in https://github.com/solvcon/modmesh/pull/553#discussion_r2229884474 , this module should be named as `_report.py`.  Do not use long names.

The contents should use that of `profiling/profile_sort.py` and the copyright belongs to the original author.